src/test/java/com/training/sample/tests/Assignment1Test.java

```diff
@@ -0,0 +1,48 @@
+package com.training.sample.tests;
+
+import com.training.sample.pages.Assignment1;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class Assignment1Test {
+    Assignment1 a1=new Assignment1();
```

Comment:
instead of a1->use assignment1 to avoid any confusion of which class object it is.